lib/bpk-component-snackbar/src/NativeSnackbar.android.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { NativeModules } from 'react-native';
+
+import { type INativeSnackbar } from './common-types';
+
+const { AndroidBpkSnackbar } = NativeModules;
+
+export const {
+  LENGTH_SHORT,
+  LENGTH_LONG,
+  LENGTH_INDEFINITE,
+} = AndroidBpkSnackbar;
```

Comment:
Is there any way we can rename these to be more explicit? 

We usually tie in the component name, for example `BUTTON_TYPES` or `BADGE_ACCESSORY_VIEW_ALIGNMENTS`. So it'd be good to make them `SNACKBAR_LENGTHS.short` and so on (note the use of plural too). It'd also be good if they were in one `SNACKBAR_LENGTHS` object too.

Adding the component name would also avoid any potential future namespace clashes if we move to requiring all components from a single Backpack import.

I realise this might be more difficult due to the bridge, I'm just assuming there's a way to map/remap these names before they go over the bridge.

Reply:
It's possible, I'm not a big fan of this type of api to be honest, it's unnecessary noise in JS. Plus if it ever clash people can alias the import. But I if this is what we generally to is better to be consistent.

Reply:
Yeah I feel like consistency is what we should go for here